src/Components/CardboardList/CardboardListItem.tsx

```diff
@@ -125,6 +125,9 @@ export const CardboardListItem = <T extends unknown>(
                         (typeof iconStart.name === 'string' ? (
                             <FontIcon
                                 iconName={iconStart.name}
+                                style={{
+                                    color: (iconStart as any).color
```

Comment:
Do you still need the cast to any with the addition of `color?: string;` in CarboardList.types.ts:41?

Reply:
yep good catch, no :) I just added null check though instead of providing a default color as you see in the updated version